.resinci.yml

```diff
@@ -2,34 +2,32 @@
 npm:
   platforms:
     - name: linux
-      os: alpine
+      os: ubuntu
```

Comment:
Replacing alpine with Ubuntu/Debian because the CLI does not even run on Alpine... Though the tests were clearly passing. The tests also take a long time to run, so I "distributed" the versions 8 / 10 / 12 among the different architectures instead of testing all 3 versions in every architecture.

Reply:
This is a little worrying, I'd much rather we ran every version on every environment, how long do the tests usually take?

Reply:
OK. Actually I think I was misunderstanding that `.resinci.yml` file, because it is configuring the `npm` job while the `repo.yml` file specifies `type: node-cli`. I am testing some changes and I have asked @nazrhom on Flowdock as well: https://www.flowdock.com/app/rulemotion/d-build-pipelines/threads/VkarI_cmcz1_gmQsyYyOaP15wzC

Reply:
Following tests, it seems that the platforms specified in `.resinci.yml` for the `npm` job type aren't actually triggering any tests. The `repo.yml` file specifies `type: node-cli`, so it is the `node-cli` job (rather than the `npm` job) that matters. If I wanted to add Node 12 to the tests, I would have to submit a PR to `resin-concourse` and update the Windows and Mac on-premises computers to install Node 12. Therefore, for this PR, I've reverted the previous `.resinci.yml` file, and I will leave any changes to it to another future PR.